src/main/python/deep_qa/layers/highway.py

```diff
@@ -0,0 +1,10 @@
+from keras.layers import Highway as KerasHighway
+
+class Highway(KerasHighway):
+    """
+    Keras's Highway layer does not support masking, but it easily could, just by returning the
```

Comment:
I'm actually thinking of going the other way, and renaming `MaskedSoftmax` to `Softmax`.  I guess my thought is, if we're fixing a bug in Keras, we'll use the same names, and maybe submit a PR to Keras, so that when if/when it gets fixed in Keras, we just change an import, and we're done.